### ISSUE TITLE
Indicator plugin (gtk 3) doesn't honor panel color in XFCE

### ISSUE THREAD

**Issue by sumurt** [snwh/paper-gtk-theme#337]
I use Xubuntu 16.04 with latest daily Paper and latest GTK:
libgtk-3-0:amd6 3.18.9-1ubun amd64
libgtk2.0-0:amd 2.24.30-1ubu amd64

Unlike Numix, Paper has different color for Indicator plugin on XFCE Panel which doesn't depend on the panel color (same for highlighing).

![screenshot_2016-05-02_09-56-01](https://cloud.githubusercontent.com/assets/12891044/14946494/31e6e63e-104d-11e6-8f48-8c0bd6c89fe6.png)

**Comment by snwh** [comment]
I'm not sure, but I think this is due to the XFCE panel being GTK2 and that indicator being GTK3.

**Comment by sumurt** [comment]
Yes, I believe it is the reason. So there must be a workaround as it works in Numix (I've tried to find it but I lack understanding of theming). Well, I just deleted the indicators applet, it was useless anyway. But it comes out of the box in Xubuntu, so this bug may affect quite a lot of people.
